index.js

```diff
@@ -13,6 +13,54 @@ connection.connect((err) => {
   }
 });
 
+app.use(express.json());
+
 app.listen(port, () => {
   console.log(`Server listening on port ${port}`);
 });
+
+app.get('/bottle', (req, res) => {
```

Comment:
good for the content, but following REST conventions the path should use plural. so `/bottles` instead of `/bottle`